docs/getting-started.md

```diff
@@ -61,12 +61,15 @@ network, a shared volume, and a service for the application.
       environment:
         - OTEL_EXPORTER_OTLP_ENDPOINT=http://<address_in_docker_network>:4318
         - OTEL_GO_AUTO_TARGET_EXE=<location_of_target_application_binary>
+        - OTEL_RESOURCE_ATTRIBUTES=key1=value1
```

Comment:
is this required to run the agent? If not, I think it is covered by the "see more variables" link below. It would be good if this example stays minimal

Reply:
Ok, I've removed that.